tests/it/arrow.rs

```diff
@@ -0,0 +1,350 @@
+use arrow2::array::*;
```

Comment:
😮  you even wrote the round trip tests. Impressive

The only thing I didn't see in here was a test for `BooleanArray`